### PR TITLE
Make mention of alternate flags in std::fmt traits

### PR DESCRIPTION
**Pull request by steveklabnik**
https://www.reddit.com/r/rust/comments/3ceaui/psa_produces_prettyprinted_debug_output/

### CODE REVIEW

**Comment by killercup** [comment]
Thanks!

One of the most awesome parts of this ist that it works for nested structures, too. Do you maybe want to add a more complex example somewhere? (Feel free to copy [my test code](http://is.gd/pWHA55)!)

**Comment by steveklabnik** [comment]
I don't know, I guess I just assume that it would do so, but maybe assumptions don't matter.

File: src/libcore/fmt/mod.rs [hunk 1]
```diff
@@ -379,6 +410,8 @@ pub trait Display {
 ///
 /// The `Octal` trait should format its output as a number in base-8.
 ///
+/// The alternate flag, `#o`, adds a `0o` in front of the output.
```
Comment (huonw) [inline src/libcore/fmt/mod.rs]: Hm, isn't just `#` the alternate flag not the full `#o` entity? (i.e. one adds the flag to the format specifier `o`)

Comment (steveklabnik) [inline src/libcore/fmt/mod.rs]: I guess technically that is true.

**Comment by steveklabnik** [comment]
@huonw updated!

**Pull request merged**